less/index.less

```diff
@@ -54,6 +54,397 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+/* Custom CSS */
+
+
+@primaryColor: #F55C42;
+@baseColor: rgb(29, 29, 29);
+@baseFont: 'Nunito', 'sans-serif';
+@sectionHFiveColor: rgba(29, 29, 29, 0.5);
+
+.flex-column {
+	display: flex;
+	flex-direction: column;
+}
+
+.flex-column-reverse {
+	display: flex;
+	flex-direction: column-reverse;
+}
+
+.flex-space-between {
+	display: flex;
+	justify-content: space-between;
+}
+
+.flex-center {
+	justify-content: center;
+	align-items: center;
+}
+
+.h1-h2-styling {
+	font-weight: bold;
+	color: @primaryColor;
+	font-size: 0.9em;
+	letter-spacing: 0.05em;
+}
+
+html {
+	background-color: rgb(109, 33, 20);
+	font-family: @baseFont;
+}
+
+.body-con {
+	margin: 0 auto;
+	max-width: 1024px;
+	background-color: rgb(255, 255, 255);
+
+	@media screen and (max-width: 500px) {
+		max-width: 450px;
+	}
+}
+
+header {
+	.flex-column();
+	padding: 2rem;
+
+	.brand-info {
+		.flex-space-between();
+
+		@media screen and (max-width: 500px) {
```

Comment:
You could have also set max-width:500px as a variable since you are using it multiple times

Reply:
I have set the mobile screen size as a variable and used it throughout the project.